### ISSUE TITLE
Package can't be removed

### ISSUE THREAD

**Issue by drbartling** [SublimeText-Markdown/MarkdownEditing#557]
Personally, I found this package to be way more than I wanted.
Having the theme change while using goto anywhere was bothersome, among other things.

So, I go to remove the package and I can't.
Reproduced on a clean install of sublime.

Installed Markdown editing and markdown preview, and neither can be removed through the package manager.

**Comment by GiovanH** [comment]
What error do you get when you try to remove the package? Is there any console output?

**Comment by rubslopes** [comment]
Hi, I have this same problem. I tried to remove it through Package Control, but it was still showing package features, theme, and syntax when I opened markdown or txt files. So I went to the package directory and deleted all folders. When I restarted Sublime, it said "7 dependencies were absent and were reinstalled" or something like that. It seems the only way is to reinstall Sublime completely.

**Comment by bencwallace** [comment]
Same issue. After removing the package through package control, if I switch the language to Markdown, I get the following errors in the console:
```
error: Error loading colour scheme Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme: Unable to open Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme
error: Error loading colour scheme Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme: Unable to open Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme
error: Error loading colour scheme Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme: Unable to open Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme
error: Error loading colour scheme Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme: Unable to open Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme
```

For now, I can avoid getting these errors by switching the syntax to MultiMarkdown.

**Comment by deathaxe** [comment]
MarkdownEditing currently saves color scheme settings in syntax specific settings, which are left untouched when uninstalling the package.

You may want to remove `color_scheme` key manually by editing _Packages/User/Markdown.sublime-settings_.

**Comment by deathaxe** [comment]
Should be resolved by 3.0.0
